src/catalogue/application/produit/__tests__/produitController.spec.ts

```diff
@@ -2,37 +2,30 @@ import fs from 'fs'
 import { makeExecutableSchema } from 'graphql-tools'
 import { graphql } from 'graphql'
 import Resolver from "../../resolverMap";
-import { CatalogueDependencyContainer } from "../../../configuration/catalogue.dependencyContainer";
-import { ServerDependencyContainer } from "../../../../configuration/serverDependencyContainer";
+import { createFakeServerDependenciesContainer } from "../../../../configuration/__tests__/serverDependencyContainer";
 
 //https://gist.github.com/nzaghini/e038ff05c60bc2c5435f8331f890cea4
-const recupererProduit = {
-    exécuter: jest.fn().mockReturnValue({
-            id: "1",
-            nom: "Pomme",
-            poids: 200,
-            prix: 1
-        }
-    )
-};
-const recupérerLesProduits = {
-    exécuter: jest.fn().mockReturnValue([{
+
+const catalogueContainer = createFakeServerDependenciesContainer().catalogueDependencyContainer
+
+// @ts-ignore
+catalogueContainer.recupererLeProduit.exécuter.mockReturnValue({
```

Comment:
```suggestion
catalogueContainer.recupererLeProduit.exécuter = jest.fn().mockReturnValue({
```
comme ça tu peux retirer le ts-ignore